kopf/on.py

```diff
@@ -145,6 +145,7 @@ def resume(  # lgtm[py/similar-function]
         labels: Optional[filters.MetaFilter] = None,
         annotations: Optional[filters.MetaFilter] = None,
         when: Optional[callbacks.WhenFilterFn] = None,
+        status_prefix: bool = True,
```

Comment:
**Important:** Generally, as a rule of thumb, all handler options are designed so that the default value is `None` and evaluates to falseness. Even for booleans (see `optional` or `deleted` in some handlers nearby — it is `Optional[bool]`). I would prefer that all user-facing DSL features follow this convention in the future.

Can you please negate the option and its meaning? (I am not sure what would be the better wording to describe this: flat status? direct status? raw status? result_to_status? something else?)